bundle/Factory/AutoResponderDataFactory.php

```diff
@@ -16,7 +16,7 @@
 use function array_key_exists;
 use function trim;
 
-class AutoResponderDataFactory extends EmailDataFactory
+class AutoResponderDataFactory extends EmailDataFactory implements EmailDataFactoryInterface
```

Comment:
No need for `implements` here, as this class gets the `implements` from base class.